### ISSUE TITLE
An http API is required so i can reserve the setup from my command line

### ISSUE THREAD

**Issue by PimClappers** [DavidBergevoet/Setup-Reservation#9]
Imagine just having to type "reserveCorn 20" in my linux command line to reserve te setup for 20 minutes. Yes please!

**Comment by InzeNL** [comment]
@DavidBergevoet would it be an idea to perform the "organising of the Python files" in this task?

**Comment by DavidBergevoet** [comment]
Yeah it would be but no other branch should be open to have no weird merges or conflicts

**Comment by InzeNL** [comment]
In regards to #14, this should also create an endpoint to obtain all available setups.
Along with that, a "reservations" query should have a filter option to filter by one or more setup ids

**Comment by InzeNL** [comment]
We should also add swagger to this
https://pypi.org/project/flask-swagger/
